### DISCUSSION TITLE
When using FDS6.7.4 version to calculate simulation, the out file encountered the following error:

### DISCUSSION THREAD

**Discussion by goodman249** [firemodels/fds#12708] [General]
When using FDS6.7.4 version to calculate simulation, the out file encountered the following error:
Time Step 100 March 29, 2024 22:09:53
Step Size: 0.103E-02 s, Total Time: 0.08 s
Pressure Iterations: 4
Maximum Velocity Error: 0.11E+00 on Mesh 12 at (11 4 80)
Maximum Pressure Error: 0.56E+05 on Mesh 7 at (10 27 1)
How to avoid Maximum Velocity Error and Maximum Pressure Error

**Comment by rmcdermo** [comment]
This is not the kind of "error" you can avoid.  The .out file is reporting the numerical errors you are encountering, which is inherent in all numerical simulations.  You can reduce the error tolerances but the code will run slower.  See the section on "Accuracy of the Pressure Solver" in the user guide.

**Comment by drjfloyd** [comment]
You cannot avoide those errors. Those are FDS reporting the current numerical error. All CFD codes are subject to numerical error.
